### PR TITLE
Build on ARM

### LINKED ISSUE #85: Add packaging for ARM
Need to add packaging for ARM and add it to the ARM docker image.

### PR DESCRIPTION
**Pull request by cevian**
Fixes #85

### CODE REVIEW

**Review by jgpruitt — DISMISSED** [review]
LGTM

**Comment by isala404** [comment]
Hi, could we get this merge? I'm working on a side project utilizing pgvectorscale, but unable to test it locally since the current ARM build of the TimescaleDB Docker image is throwing an error that the extension is not found.

File: .github/workflows/deb-packager.yaml [hunk 1]
```diff
@@ -6,25 +6,32 @@ on:
...
+          - major: 15
+            minor: 7
+          - major: 16
+            minor: 3
+        platform:
+          - type: amd64
+            runs_on: ubuntu-latest
+            rustflags: '-C target-feature=+avx2,+fma'
+          - type: arm64
+            runs_on: cloud-image-runner-arm64
```
Comment (adolsalamanca) [inline .github/workflows/deb-packager.yaml:31]: Where's this image coming from?

**Review by adolsalamanca — APPROVED** [review]
Looks good, just left a tiny comment

**Review by jgpruitt — APPROVED** [review]
Did not run, but makes sense to me. Didn't spot anything.

File: .github/workflows/deb-packager.yaml (hunk 1 above)
Comment (adolsalamanca) [reply .github/workflows/deb-packager.yaml:31]: Found it

**Comment by cevian** [comment]
note the run https://github.com/timescale/timescaledb-docker-ha/pull/472 to build into docker.

**Pull request merged**